src/rules/file/hasDoxygenHeader.ts

```diff
@@ -37,9 +55,12 @@ const test = (value: string) => {
 }
 
 const fix = (value: string, config?: LintConfig): string => {
-  if (test(value).length === 0) {
+  const result = test(value, config)
+  if (result.length === 0) {
     return value
-  }
+  } else if (result[0].message == messageForSingleAsterisk)
+    return value.replace('/*', '/**')
```

Comment:
this would only replace it once right, at the very start?  not all instances?

Reply:
Yes, it will replace just first one.